lib/RTCSession.js

```diff
@@ -278,7 +278,11 @@ module.exports = class RTCSession extends EventEmitter
     this._rtcAnswerConstraints = options.rtcAnswerConstraints || null;
 
     this._data = options.data || this._data;
-
+    // Check Authorization: Bearer.
+    if (this._ua.configuration.auth && extraHeaders.filter((extraHeader) => extraHeader.startsWith('Authorization: Bearer')).length==0)
```

Comment:
This is not ok:

* We must not mandate SIP headers in so strict low/upper case.
* Anyway, it looks like in order to enable JWT auth, the user must call `ua.call()` with "Authorization: Bearer" in `options.extraHeaders`.  No way, this is a hack. Let's create a different `option.useJwtAuth: boolean` instead.